### PR TITLE
Create new blog for DIVA Team at SHH

### PR DESCRIPTION
**Pull request by Ayaz1997**

### CODE REVIEW

File: packages/website/_posts/diva-team-at-solana-hacker-house.mdx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: DIVA team at Solana Hacker House
+description: Sharing with you few lines of experience from the Solana Hacker House event representing DIVA at Solana Hacker House, Bangalore.
+date: 10-14 May 2022
```
Comment (juliankrispel) [inline packages/website/_posts/diva-team-at-solana-hacker-house.mdx]: this needs to be the correct date format, please copy what's in the other blogpost

File: packages/website/_posts/diva-team-at-solana-hacker-house.mdx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+coverImage: 888whales.png
```
Comment (juliankrispel) [inline packages/website/_posts/diva-team-at-solana-hacker-house.mdx]: you need to change the cover image or remove it. Do you have an image of you guys?

File: packages/website/_posts/diva-team-at-solana-hacker-house.mdx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+coverImageDescription: A collection of whales
```
Comment (juliankrispel) [inline packages/website/_posts/diva-team-at-solana-hacker-house.mdx]: image description doesn't make sense for the post

**Review by juliankrispel — CHANGES_REQUESTED** [review]
Hey @Ayaz1997 - this mostly looks good but the frontmattter block at the top has some mistakes which causes the build to fail

File: packages/website/_posts/diva-team-at-solana-hacker-house.mdx (hunk 1 above)
Comment (Ayaz1997) [reply packages/website/_posts/diva-team-at-solana-hacker-house.mdx]: Okay Changing it

File: packages/website/_posts/diva-team-at-solana-hacker-house.mdx (hunk 2 above)
Comment (Ayaz1997) [reply packages/website/_posts/diva-team-at-solana-hacker-house.mdx]: Updated the changes requested

File: packages/website/_posts/diva-team-at-solana-hacker-house.mdx (hunk 3 above)
Comment (Ayaz1997) [reply packages/website/_posts/diva-team-at-solana-hacker-house.mdx]: Updated the changes requested

**Review by juliankrispel — APPROVED** [review]
🎉

**Pull request merged**